managed/services/management/alerting/service.go

```diff
@@ -942,3 +987,36 @@ func validateParameters(definitions models.AlertExprParamsDefinitions, values mo
 var (
 	_ alerting.AlertingServer = (*Service)(nil)
 )
+
+// AlertExprParamValue represents rule parameter value.
+type AlertExprParamValue struct {
+	Name        string           `json:"name"`
+	Type        models.ParamType `json:"type"`
+	BoolValue   bool             `json:"bool"`
+	FloatValue  float64          `json:"float"`
+	StringValue string           `json:"string"`
+}
+
+// AlertExprParamsValues represents rule parameters values slice.
+type AlertExprParamsValues []AlertExprParamValue
+
+// AsStringMap convert param values to string map, where parameter name is a map key and parameter value is a map value.
+func (p AlertExprParamsValues) AsStringMap() map[string]string {
+	m := make(map[string]string, len(p))
+	for _, rp := range p {
+		var value string
+		switch rp.Type {
+		case models.Float:
+			value = fmt.Sprint(rp.FloatValue)
```

Comment:
I wonder how can this pass the `forbidigo` linter rule, which has been [enabled](https://github.com/percona/pmm/pull/2302/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9L118) not long ago? Is the linter failing to detect it here or is it misconfigured by any chance?

Reply:
Isn't it meant to catch instances of `fmt.Print*` (i.e., print debugging) and not `fmt.Sprint*`?

Reply:
Oh yes, that's the default configuration :) Thanks!